src/main/java/org/example/PhoneNumberValidator.java

```diff
@@ -0,0 +1,9 @@
+package org.example;
+
+public class PhoneNumberValidator {
```

Comment:
携帯を意味するMobileを入れておくとbetterですね！
```suggestion
public class MobilePhoneNumberValidator {
```